package.json

```diff
@@ -11,19 +11,11 @@
     "url" : "https://github.com/prebid/Prebid.js.git"
   },
   "adapters": [
```

Comment:
@mkendall07 with this commit and by removing bidders here we can demo rendering ads as soon as bids are back. With all bidders the setTimeout always is reached due to testing config differences between bidders.